frontend/src/components/games/themes/GameGrid.css

```
@@ -1,128 +1,51 @@
+  /* Much more conservative CSS fallback to ensure full grid visibility */
+  --ui-space: 360px; /* Increased for safety */
+  --scrollbar: 20px;
+  --available-height: calc(100vh - var(--ui-space));
+  --available-width: calc(100vw - var(--scrollbar) - 60px);
+  
+  /* Use smaller dimension with much more conservative maximum */
+  width: min(var(--available-height), var(--available-width), 480px);
+  height: min(var(--available-height), var(--available-width), 480px);
+  
+  /* JavaScript will override with precise calculation via --calculated-size */
+  width: var(--calculated-size, min(var(--available-height), var(--available-width), 480px));
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)